### DISCUSSION TITLE
y轴最大值自适应问题

### DISCUSSION THREAD

**Discussion by 18211365467** [antvis/G2#5682] [Q&A] [answered]
- **G2 Version**:
- **Platform**:
- **Mini Showcase(like screenshots)**:
- **CodePen Link**:

**Comment by 18211365467** [comment]
![image](https://github.com/antvis/G2/assets/79558944/045a58a0-1810-41ae-8d5f-11d099ddbf40)

**Comment by 18211365467** [comment]
大于15怎么自适应线上兼容

**Comment by hustcc** [comment, marked as answer]
```ts
.scale('y', { nice: true })
```

这种场景反正有利有弊，按需选择。
